tests/unit/objects/test_statistics.py

```diff
@@ -0,0 +1,49 @@
+"""
+GitLab API: https://docs.gitlab.com/ee/api/statistics.html
+"""
+
+import pytest
+import responses
+
+content = {
+    "forks": "10",
+    "issues": "76",
+    "merge_requests": "27",
+    "notes": "954",
+    "snippets": "50",
+    "ssh_keys": "10",
+    "milestones": "40",
+    "users": "50",
+    "groups": "10",
+    "projects": "20",
+    "active_users": "50",
+}
+
+
+@pytest.fixture
```

Comment:
Would it make sense to move the fixture code to conftest.py file?

Reply:
I'm not sure if any other unit tests will need this particular fixture, I'm following the same pattern as other unit tests

Reply:
I also think in this case it makes sense to keep the fixtures here, close to the scope where they are used as we do with other endpoint-specific fixtures: